resize-observer/src/main.ts

```diff
@@ -0,0 +1,19 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import ResizeObserver from 'resize-observer-polyfill';
+import global from '@dojo/shim/global';
+import App from './widgets/App';
+import '@dojo/shim/browser';
+
+if (!global.ResizeObserver) {
+	global.ResizeObserver = ResizeObserver;
+}
+
+// Create a projector to convert the virtual DOM produced by the application into the rendered page.
```

Comment:
let's remove the comments from create app

Reply:
Comments removed